scripts/.env

```diff
@@ -20,5 +19,19 @@ export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:/usr/share/gazebo-11/models:
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib/x86_64-linux-gnu/gazebo-11/plugins
 
+# GZ SIM
```

Comment:
Do we need this lines?

Reply:
@Arcane-01 just did some tests and we can remove those lines, hooks are enough to find every resource